cmd/argocd-application-controller/commands/argocd_application_controller.go

```diff
@@ -233,6 +237,9 @@ func NewCommand() *cobra.Command {
 	command.Flags().BoolVar(&enableDynamicClusterDistribution, "dynamic-cluster-distribution-enabled", env.ParseBoolFromEnv(common.EnvEnableDynamicClusterDistribution, false), "Enables dynamic cluster distribution.")
 	command.Flags().BoolVar(&serverSideDiff, "server-side-diff-enabled", env.ParseBoolFromEnv(common.EnvServerSideDiff, false), "Feature flag to enable ServerSide diff. Default (\"false\")")
 	command.Flags().DurationVar(&ignoreNormalizerOpts.JQExecutionTimeout, "ignore-normalizer-jq-execution-timeout-seconds", env.ParseDurationFromEnv("ARGOCD_IGNORE_NORMALIZER_JQ_TIMEOUT", 0*time.Second, 0, math.MaxInt64), "Set ignore normalizer JQ execution timeout")
+	// argocd k8s event logging flag
+	command.Flags().StringSliceVar(&enableK8sEvent, "enable-k8s-event", env.StringsFromEnv("ARGOCD_ENABLE_K8S_EVENT", []string{"all"}, ","), "Enable ArgoCD k8s event. use k8s argocd event reason, to disable all events, use the `none`. (e.g --enable-k8s-event=StatusRefreshed,ResourceCreated or --enable-k8s-event=none)")
```

Comment:
What do you think about below description?

```suggestion
	// argocd k8s event logging flag
	command.Flags().StringSliceVar(&enableK8sEvent, "enable-k8s-event", env.StringsFromEnv("ARGOCD_ENABLE_K8S_EVENT", []string{"all"}, ","), "Enable ArgoCD to use k8s events. For disabling all events, set the value as `none`. (e.g --enable-k8s-event=StatusRefreshed,ResourceCreated or --enable-k8s-event=none)")
```

Reply:
@ishitasequeira 
```suggestion
	command.Flags().StringSliceVar(&enableK8sEvent, "enable-k8s-event", env.StringsFromEnv("ARGOCD_ENABLE_K8S_EVENT", []string{"all"}, ","), "Enable ArgoCD to use k8s event. For disabling all events, set the value as `none`. (e.g --enable-k8s-event=none), For enabling specific events, set the value as `event reason`. (e.g --enable-k8s-event=StatusRefreshed,ResourceCreated)")
```
how about that?

Reply:
Looks good.